README.md

```diff
@@ -1,19 +1,63 @@
 # HWSC-File-Transaction-SVC
 This is one of the microservices in the Humpback Whale Social Call project, which allows user to manage their files. 
-### Purpose
 
+### Purpose
 This provides file management service which includes the following:
-  - Downloading
-  - Uploading
-  - Deleting
+- Downloading
+- Uploading
+- Deleting
+
 ## Prerequisites
-- Python 3.7
+- Python 3.7 or greater
+- pip
+- pipenv
 - Webstorm
 
-## How to install Dependencies
- - TO-DO
+## Check your python version
+- On your terminal, type [python --version] to check the python version
```

Comment:
remove "to check the python version", the header says what the command does. Also I think it might be better to swap all the brackets with back ticks (``) and start the command with the $:

- Open your terminal and type `$ python --version`